src/pages/steps/RichAlert.tsx

```diff
@@ -0,0 +1,29 @@
+import { Alert } from '@/ui/Alert';
+import { Box, parseSpacing } from '@/ui/Box';
+
+function RichAlert({ children, ...props }) {
```

Comment:
Type the props with the `Props` type from `Alert` the alert component

We also tend to stick to arrow functions everywhere in this project
```suggestion
const RichAlert = ({ children, ...props }: AlertProps) => {
```